template_config.yml

```diff
@@ -25,6 +25,12 @@ plugin_name: pulp_rpm
 plugin_snake: pulp_rpm
 publish_docs_to_pulpprojectdotorg: true
 pulp_settings:
+  allowed_content_checksums:
+  - "sha1"
```

Comment:
@dralley, do we need md5 as well here or do we want to fail if we test any md5 content?

Reply:
I'd prefer to fail

Reply:
Works for me